src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Payment/PaymentAuthorizationTests.cs

```diff
@@ -3,6 +3,8 @@
 using Sample.AspNetCore.SystemTests.Services;
 using Sample.AspNetCore.SystemTests.Test.Helpers;
 using SwedbankPay.Sdk;
+using SwedbankPay.Sdk.Common;

```

Comment:
Please get rid of the `Common` namespace by moving stuff within it up one level in the namespace hierarchy.

```suggestion
```